Microsoft.Toolkit/Microsoft.Toolkit.csproj

```diff
@@ -14,14 +14,14 @@
     <PackageTags>Windows;Community;Toolkit;WCT;UWP;Incremental;Loading;Collection;IncrementalLoadingCollection;String;Array;extensions;helpers</PackageTags>
   </PropertyGroup>
 
-  <!-- .NET Standard 1.4 doesn't have the [Pure] attribute -->
-  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard1.4' ">
-    <PackageReference Include="System.Diagnostics.Contracts" Version="4.3.0" />
-  </ItemGroup>
-
   <!-- .NET Standard 2.1 and .NET 5 already have [NotNullIfNotNull] and [NotNullWhen] -->
-  <PropertyGroup Condition=" '$(TargetFramework)' == 'netstandard2.1' OR '$(TargetFramework)' == 'net5.0' ">
+  <PropertyGroup Condition="'$(TargetFramework)' == 'netstandard2.1' OR '$(TargetFramework)' == 'net5.0'">
     <DefineConstants>NETSTANDARD2_1_OR_GREATER</DefineConstants>
   </PropertyGroup>
 
+  <!-- .NET Standard 1.4 doesn't have the [Pure] attribute -->
+  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard1.4'">
```

Comment:
I'm not claiming anything. I made these changes **because** it was difficult for **me** to read.

I got the method to madness, when I looked at one or more project files. I knew how it was arranged. See, when you order by a property value, you'll have duplicate entries, when there are same properties or items with multiple conditions. You'll also have a mix of properties and items trail by each condition. It'll look fine with single condition but with overlapping and multiple conditions, makes the project file not only harder to read, but also harder to reason about.

I think you see the project file like a C# source where the conditional grouping is similar to pre-processor defines. That's why, this ordering makes sense to you and mine is bit harder.

MSBuild project file is different from sources, it's an **XML** file having specific element schema of properties, items, imports and targets. It won't scale very well, if we group by property value. But grouping via element types always work very well and scales very well too. It is consistent if you look at the elements order instead of a conditional order.

_That being said, I'll try to preserve the ascending order for `TargetFramework` condition. Instead of sorting them globally, grouping by element type and then sorting by `TF` would yield similar effect._

**Also, this is just clean-up of existing lines without actual refactoring. Please see #3894 for the rest of the changes.**